pkg/db/strategy.go

```diff
@@ -392,6 +396,9 @@ func (s *Strategy) update(ctx context.Context, status bool, obj types.Object) (t
 func (s *Strategy) Create(ctx context.Context, obj types.Object) (result types.Object, err error) {
 	err = s.db.Transaction(ctx, func(ctx context.Context) error {
 		result, err = s.create(ctx, obj)
+		if err, ok := err.(*mysql.MySQLError); ok && err.Number == 1062 {
+			return newConflict(s.gvk, obj.GetName(), err)
+		}
```

Comment:
Any reason not to use `translateDuplicateEntryErr` here?

Reply:
I had forgotten it there. Fixed